go/types/list.go

```diff
@@ -175,7 +206,10 @@ func (l List) Iter(f listIterFunc) {
 
 type listIterAllFunc func(v Value, index uint64)
 
+// IterAll iterates over the list and calls f for every element in the list. Unlike Iter there is no
+// way to stop the iteration and all elements are visited no matter what.
```

Comment:
I would leave out "and all elements are visited no matter what", or at least the "no matter what".